javascript-typing-tutor/script.js

```diff
@@ -21,24 +24,38 @@ document.addEventListener('keydown', function (e) {
       $background.className = 'background-color-off';
       $opacity.className = 'opacity text';
       $modalText.textContent = Math.round(value / attempts * 100) + '% with ' + (attempts - value) + ' errors.';
-      $retry2.addEventListener('click', function () {
-        if (test === true) {
-          location.reload();
-        }
-      });
     }
   } else {
     $text[value].className = 'wrong-text under-line';
     attempts++;
   }
+}
+
+function modalRetry(e) {
+  if (e) {
```

Comment:
This conditional is likely not necessary 🤔 

When would you not get an event object? This seems like it would always trigger the conditional.